src/wwwroot/services/auth.js

```diff
@@ -37,25 +37,41 @@
       isUrlAllowed: isUrlAllowed,
       getDefaultUrl: getDefaultUrl
     };
-
+    var loginSession = null;
     var decodedToken = null;
     var encodedToken = null;
     var temporarilyAuthed = false;
 
-    encodedToken = $window.localStorage.getItem('jwtToken');
-    if (encodedToken) {
-      decodeToken();
-      // verify expiration
-    }
+    init();
     return authService;
 
+    function init() {
+      loginSession = getLoginSession('relayLogin');
+      encodedToken = loginSession? loginSession.jwtToken : null;
+      if (encodedToken) {
+        decodeToken();
+        // verify expiration
+      }
+    }
+
+    function getLoginSession(storageName) {
+      var login = angular.fromJson($window.localStorage.getItem(storageName));
+      return login;
+    }
+
     // Save the token in the local storage (globally) or in a temporal variable (local)
     function saveToken(token, isTemporal) {
       encodedToken = token;
       decodeToken();
       temporarilyAuthed = !!isTemporal;
       if (!temporarilyAuthed) {
-        $window.localStorage.setItem('jwtToken', token);
+        loginSession = {
+          jwtToken: token,
```

Comment:
What do you think about renaming it as _token_?

Reply:
Ok, I think is fine.